source/base/Text.ooc

```diff
@@ -182,6 +182,16 @@ Text: cover {
 		printf("%s\n", this _buffer _backend pointer as Char*)
 		this free(Owner Receiver)
 	}
+	format: func (args: ...) -> This {
+		t := this take()
+		string := t toString()
+		resultString := string format(args)
+		string free()
+		result := This new(resultString) copy()
+		resultString free()
+		this free(Owner Receiver)
+		result give()
```

Comment:
This `give()` could be moved to the end of the line where you assign `result`, but it's not important.